hawc/apps/common/helper.py

```diff
@@ -204,152 +192,5 @@ def _get_tags(cls, e):
         return returnValue
 
     def build_response(self):
-        header_row = self._get_header_row()
```

Comment:
ohh yeah! this is a good screenshot:
![Screen Shot 2020-04-14 at 8 42 47 PM](https://user-images.githubusercontent.com/999952/79287055-90537380-7e90-11ea-81e1-123bd577cf1e.png)